docs/using-airbyte/mappings.md

```diff
@@ -64,44 +66,44 @@ There are many reasons you might want to filter rows.
 - **Optimization**: Smaller datasets require less processing and storage, and you can query them faster.
 - **Compliance**: You don't want to keep data longer than is needed for a defined business purpose, and you want to ensure you don't accidentally archive data from individuals who opted out of data collection.
 
+:::note
+You can only filter fields whose type is string and number.
+:::
+
+## Considerations and limitations
+
+Before you begin mapping fields, consider the following.
+
+- If you set up mapping on an existing connection, Airbyte prompts you to clear your stream and the data in your destination. You almost certainly should do this. Not doing this could result in undesirable or unexpected behavior. For example, if you set up hashing on a field that contains personally identifiable information (PII), you want all data in that column to be hashed, not just new data. Clearing your destination accomplishes this. However, if you're dealing with an extremely large database, a complete resync might take a significant amount of time.
+- Do not map a cursor or primary key field if the [sync mode](core-concepts/sync-modes) depends on it. Doing this disrupts Airbyte's ability to use them for incremental syncing and deduplication.
```

Comment:
@pedroslopez did we decide that this would actually be disruptive or not?  I can't recall.